### DISCUSSION TITLE
Does this work with xcodeproj  or xcworkspace folder structures to resolve spm dependencies?

### DISCUSSION THREAD

**Discussion by teejayhh** [swiftlang/vscode-swift#776] [Q&A] [answered]
Hello, I am currently trying to replace appcode (intellyj) with something else, which is not Xcode (I have my reasons). For simple packages `vscode-swift` seems perfectly fine but if I open an iOS app project folder I run into issues that the interpreter has no clue of the apps dependencies and can't provide any completions. Tools like 'sweetpad' make an alternative xcode dev environment  sooooo close to viable if there wasn't this issue. 

I assume this vscode package is not meant to parse the project file but I just wanted to ask. 
Cheers Thomas

**Comment by adam-fowler** [comment, marked as answer]
There is no support for loading Xcode project or workspace files and we aren't planning any. This thread on the forums might be of interest though https://forums.swift.org/t/wip-vscode-xcodegen-extension/69236

**Reply by teejayhh** [reply]
Thanks for your answer. By the looks those 2 links also dont tackle the spm issues I am having. I guess Xcode is the only way.
